paparazzi-preview-processor/src/main/java/app/cash/paparazzi/preview/processor/PaparazziPoet.kt

```diff
@@ -0,0 +1,144 @@
+package app.cash.paparazzi.preview.processor
+
+import com.google.devtools.ksp.getVisibility
+import com.google.devtools.ksp.symbol.KSFunctionDeclaration
+import com.google.devtools.ksp.symbol.KSValueParameter
+import com.google.devtools.ksp.symbol.Visibility
+import com.squareup.kotlinpoet.CodeBlock
+import com.squareup.kotlinpoet.FileSpec
+import com.squareup.kotlinpoet.buildCodeBlock
+
+internal object PaparazziPoet {
+  fun buildFiles(
+    functions: Sequence<KSFunctionDeclaration>,
+    isTest: Boolean,
+    env: EnvironmentOptions
+  ) =
+    if (isTest) {
```

Comment:
does this imply tests will be codegen for test previews as well?  if so, perhaps that's too much scope and we should limit to main source tests only?

Reply:
sorry this implies that we only want to generate preview codegen in the main source set